src/equations/compressible_navier_stokes_1d.jl

```diff
@@ -117,7 +118,8 @@ function CompressibleNavierStokesDiffusion1D(equations::CompressibleEulerEquatio
     CompressibleNavierStokesDiffusion1D{typeof(gradient_variables), typeof(gamma),
                                         typeof(mu),
                                         typeof(equations)}(gamma, inv_gamma_minus_one,
-                                                           mu, Prandtl, kappa,
+                                                           mu, Prandtl,
+                                                           kappa, max(1, kappa),
```

Comment:
Does this lead to type ambiguity because you compare an `Int` with a `Real`? Does the `max` function promote and return a `Real`?

Reply:
Should be fine:

```julia
julia> @code_warntype max(1, 2.0)
MethodInstance for max(::Int64, ::Float64)
  from max(x::Real, y::Real) @ Base promotion.jl:471
Arguments
  #self#::Core.Const(max)
  x::Int64
  y::Float64
Body::Float64
1 ─ %1 = Base.max::Core.Const(max)
│   %2 = Base.promote(x, y)::Tuple{Float64, Float64}
│   %3 = Core._apply_iterate(Base.iterate, %1, %2)::Float64
└──      return %3
```